examples/metalk8s-solution-example/ui/src/translations/fr.json

```diff
@@ -5,19 +5,14 @@
   "submit": "Connexion",
   "about": "À propos",
   "log_out": "Déconnexion",
-  "create_new_customResource": "Créer un nouveau Custom Resource",
-  "edit_customResource": "Éditer un Custom Resource",
   "operator_version": "Version de l'opérateur",
   "name": "Nom",
   "cancel": "Annuler",
   "create": "Créer",
   "edit": "Éditer",
-  "namespace": "Namespace",
   "version": "Version",
   "replicas": "Duplication",
   "no_data_available": "Pas de données disponibles",
-  "custom_resource": "Custom Resource",
-  "select_a_namespace": "Sélectionner un namespace…",
   "not_found": "Aucun résultat",
   "not_valid_version": "Ce n'est pas une version valide"
 }
```

Comment:
Seems like we are missing French translations for the newly added texts?

Reply:
Yes. because I18n is not implemented in the UI yet.